src/obfl-specification.html

```diff
@@ -1823,6 +1823,16 @@ <h4 id="L84911" class="include">volume-template</h4>
             number. It can be used in an expression, e.g. <code>(= $volume
             1)</code>.</dd>
           <dd>Default is 'volume'.</dd>
+        <dt>started-volume-first-content-page-number-variable [optional]</dt>
+          <dd>Definition of the name of the variable holding the first page
+            number of the content of the started volume. It can be used in an
+            expression, e.g.
+            <code>$started-volume-first-content-page</code>.</dd>
+          <dd class="markup note">This is not necessary the first page number
```

Comment:
I would go further and say it has no relation at all with the current volume.

Reply:
Done.